packages/frontend/src/pages/EventDetails.tsx

```
@@ -0,0 +1,321 @@
+    features: [
+      'Loge privative',
+      'Service personnalisé',
+      'Restauration gastronomique',
+      'Accès coulisses',
+      'Parking VIP',
+    ],
+  },
+];
+
+// TODO: À remplacer par le vrai context du panier
+const useCart = () => {
```

ESLint

[error] 'useCart' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)